### PR TITLE
Adding sidecar feature to ZookeeperOperator

### PR DESCRIPTION
**Pull request by adapured**
Change log description

Adding sidecar features to zookeeper-operator

Purpose of the change

Integrate. and enable the Sidecars with zookeeper-operator

What the code does

Enable and. configure volumes and environment variables required for sidecars.

How to verify it

1.Helm lint.
2.Update the .Values with Any Sidecar containers.
3.By adding required additional volumes and Environment variables.
4.helm template.

### CODE REVIEW

**Comment by anishakj** [comment]
@adapured  could you please resolve conflict?

**Comment by anishakj** [comment]
> @adapured could you please resolve conflict?

It looks like your branch is not forked from `pravega/zookeeper-operator`

**Comment by adapured** [comment]
> @adapured could you please resolve conflict?

@anishakj Resolved the conflicts.

Thanks,
Adapured

**Comment by adapured** [comment]
> > @adapured could you please resolve conflict?
> 
> It looks like your branch is not forked from `pravega/zookeeper-operator`

**_git remote -vvv_**
origin	https://github.com/adapured/zookeeper-operator.git (fetch)
origin	https://github.com/adapured/zookeeper-operator.git (push)
upstream	https://github.com/pravega/zookeeper-operator.git (fetch)
upstream	https://github.com/pravega/zookeeper-operator.git (push)

Resolved the conflicts.

**Comment by adapured** [comment]
Fork needs to be fixed. Hence closing this PR and opening new one.

File: charts/zookeeper-operator/templates/operator.yaml [hunk 1]
```diff
@@ -39,10 +43,16 @@ spec:
...
+        {{- if .Values.additionalEnv }}
+{{ toYaml .Values.additionalEnv | indent 8 }}
+        {{- end }}
         {{- if .Values.resources }}
         resources:
 {{ toYaml .Values.resources | indent 10 }}
         {{- end }}
+      {{- if .Values.additionalSidecars }}
+{{- include "chart.additionalSidecars" . | indent 6 }}
+      {{- end }}
```
Comment (SrishT) [inline charts/zookeeper-operator/templates/operator.yaml:55]: could we also add `.Values.additionalSidecars` and `.Values.additionalVolumes` in the same way in which we are adding `.Values.additionalEnv`, without having to define it in the helpers file?

**Pull request closed without merging**